tools/deploy.py

```diff
@@ -0,0 +1,27 @@
+"""Deploys a validated version of All of Us terminology to the RDR."""
+
+import logging
+
+from client import Client
+from main_util import configure_logging, get_parser
+
+
+_HEAD_VERSION = 'HEAD'
+
+
+def deploy_terminology(rdr_project, terminology_git_version):
+  _check_out_version(terminology_git_version)
+  client.request_json('POST', _build_payload())
+
+
+if __name__ == '__main__':
+  configure_logging()
+  parser = get_parser()
+  parser.add_argument(
+      'project', default='all-of-us-rdr-sandbox',
+      help='RDR environment/project name.')
```

Comment:
Does `all-of-us-rdr-sandbox`exist? I'm not sure I have access if so (or maybe that's the point)?